flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/failover/flip1/ExecutionFailureHandlerTest.java

```diff
@@ -40,24 +42,38 @@
  */
 public class ExecutionFailureHandlerTest extends TestLogger {
 
+	private static final long restartDelayMs = 1234L;
```

Comment:
The convention for constants is to uppercase them, e.g., https://www.oracle.com/technetwork/java/codeconventions-135099.html (see on the bottom of the page)

Reply:
Fixed.